### PR TITLE
Add test of compiling a table with 144 columns

### PR DESCRIPTION
**Pull request by d6y**
Re: #22 

E.g.,

```
$ JAVA_OPTS="-Xss4M" sbt
sbt> clean
sbt> test:compile
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ scala:
 - 2.11.12
 - 2.12.4
 
+sbt_args: -J-Xss4M
```
Comment (d6y) [inline .travis.yml]: I don't think this is doing anything.  The build ran with `Xss6M` by the look of it:

https://travis-ci.org/underscoreio/slickless/jobs/357367912

**Comment by d6y** [comment]
Closing - this test doesn't add anything to existing tests.

**Comment by NigeWarren** [comment]
@d6y Hi Richard - Just out of interest - did you close because the coverage didnt change or did you close because it failed to find an extreme case already exercised but not at magnitude. Genuinely interested. N

**Comment by NigeWarren** [comment]
@d6y Great thanks Richard - I understand that both of my assumptions were unfounded hence why it was intriguing.

**Pull request closed without merging**